README.md

```diff
@@ -1,5 +1,14 @@
 # Puppet Module for zram_generator
 
+[![Build Status](https://github.com/voxpupuli/puppet-zram_generator/workflows/CI/badge.svg)](https://github.com/voxpupuli/puppet-zram_generator/actions?query=workflow%3ACI)
+[![Release](https://github.com/voxpupuli/puppet-zram_generator/actions/workflows/release.yml/badge.svg)](https://github.com/voxpupuli/puppet-zram_generator/actions/workflows/release.yml)
+[![Puppet Forge Version](http://img.shields.io/puppetforge/v/puppet/zram_generator.svg)](https://forge.puppetlabs.com/puppet/zram_generator)
+[![Puppet Forge Downloads](http://img.shields.io/puppetforge/dt/puppet/zram_generator.svg)](https://forge.puppetlabs.com/puppet/zram_generator)
+[![Puppet Forge Endorsement](https://img.shields.io/puppetforge/e/puppet/zram_generator.svg)](https://forge.puppetlabs.com/puppet/zram_generator)
+[![puppetmodule.info docs](http://www.puppetmodule.info/images/badge.png)](http://www.puppetmodule.info/m/puppet-zram_generator)
+[![Apache v2 License](https://img.shields.io/github/license/voxpupuli/puppet-zram_generator.svg)](LICENSE)
+[![Donated by Camptocamp](https://img.shields.io/badge/donated%20by-CERN-0033a0.svg)](#transfer-notice)
```

Comment:
@traylenator I guess it's not coming from camptocamp :)

Reply:
Correct origin this time.